docs/actions-pipelines.md

```diff
@@ -5,7 +5,7 @@ This section covers how to develop, run, and test your code to ensure it will wo
 
 ## Project pipelines
 
-The [cohortextractor](actions-cohortextractor.md) section describes how to make an action which generate dummy datasets based on the instructions [defined in your `study_definition.py` script](study-def.md).
+The [ehrQL](/ehrql/) documentation describes how to make an action which generate dummy datasets based on the instructions defined in your `dataset_definition.py` script.
```

Comment:
suggestion: Maybe linking to the dummy data page for ehrQL dataset definitions would be better?

```suggestion
The [ehrQL](/ehrql/how-to/dummy-data.md) documentation describes how to make an action which generate dummy datasets based on the instructions defined in your `dataset_definition.py` script.
```

(I've not checked the correctness of that relative link, but the page is better than linking to a more general part of the ehrQL documentation, I think.)